### PR TITLE
Least common multiple inequality lemma

### PR DESCRIPTION
**Pull request by metakunt**

### CODE REVIEW

**Comment by metakunt** [comment]
I will update the description once I have used this result for a different inequality.

**Review by icecream17 — APPROVED** [review]

File: set.mm [hunk 1]
```diff
@@ -633441,8 +633548,7 @@ fixed reference functional determined by this vector (corresponding to
     3factsumint.6 $e |- ( ( ph /\ k e. B ) -> G e. CC ) $.
     3factsumint.7 $e |- ( ( ph /\ k e. B ) -> ( x e. A |-> H ) e.
      ( A -cn-> CC ) ) $.
-    $( First equation in lcm inequality proof, Theorem 3.1 from
-       ~ https://www3.nd.edu/%7eandyp/notes/AKS.pdf (Contributed by metakunt,
```
Comment (avekens) [inline set.mm:633445]: why is this feference removed? was it wrong?

Comment (icecream17) [reply set.mm:633445]: It is correctly the first equation/step in the proof of theorem 3.1, but it seems the reference moved to ~ lcmineqlem below which is the entirety of theorem 3.1

Comment (metakunt) [reply set.mm:633445]: That's correct. The theorem is now finished and I have moved the reference to the final step.

**Review by avekens — APPROVED** [review]

**Pull request merged**